BE/Domain/KMS.Product.Ktm.Entities/Models/Employee.cs

```diff
@@ -12,9 +12,10 @@ public class Employee : BaseEntity
         public string Email { get; set; }
         public string SlackAccount { get; set; }
         public DateTime JoinedDate { get; set; }
-        public int EmployeeRoleId {get; set;}
+        public int EmployeeRoleId { get; set; }
 
         public virtual EmployeeRole EmployeeRole { get; set; }
         public virtual ICollection<EmployeeTeam> employeeTeams { get; set; }
+        public string SlackUserId { get; set; }
```

Comment:
[The comment is not about this class]
I see in the API project has model folder, I think it should be in Entities project.